srcpkgs/cloudflare-warp/template

```diff
@@ -0,0 +1,29 @@
+# Template file for 'cloudflare-warp'
+pkgname=cloudflare-warp
+version=2023.3.398
+revision=1
+archs="i686 x86_64"
+short_desc="Cloudflare Warp Client"
+maintainer="Andrew Shteren <dank.txt@gmail.com>"
+license="Unknown"
+homepage="https://1.1.1.1/"
+repository="nonfree"
+restricted="yes"
+nostrip="yes"
+
+_debpkgid="cloudflare_warp_2023_3_398_1_amd64_002e48d521.deb"
```

Comment:
`${version//./_}` instead of hard coding the version, maybe also add the `1` to the version.